### DISCUSSION TITLE
Button Width

### DISCUSSION THREAD

**Discussion by amyodea** [twilio-labs/paste#3122] [General] [answered]
Hey Paste team 👋🏼  With the new One Twilio theme, is there any ability to make a button the full width of the container and not just based on the width of the copy within? Thank you!

**Comment by loreina** [comment, marked as answer]
Hi @amyodea,

In code, you can use Button's `fullWidth` prop like `<Button fullWidth></Button>` and the width will match its container. This should work across any theme in Paste.

In Figma, it's a bit trickier. Using a Button instance, you'll want to select the `Button`, `Button content  ↓`, and `Content ↓` layers, then set the horizontal sizing from `hug contents` to `fill container`. 

Hope this helps!

**Reply by amyodea** [reply]
Amazing, thank you Loreina!
